cleanlab_studio/studio/trustworthy_llm.py

```diff
@@ -0,0 +1,53 @@
+from typing import cast, Literal, TypedDict
+from cleanlab_studio.internal.api import api
+
+
+QualityPreset = Literal["best", "high", "medium", "low", "base"]
+
+
+class TlmResponse(TypedDict):
+    response: str
+    confidence_score: float
```

Comment:
Base preset doesn't return this, so this return type is a bit of a lie.

Reply:
base preset does return `confidence_score`, value will just be `nan`